src/components/AddNote.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import { Icon } from 'semantic-ui-react';
```

Comment:
Here is a point.

Lets say you have a package called X. it has a sub-package called Y. You want K from Y. Never import the whole X just import Y.K to increase your website loading speed

Reply:
I didn't get it :(((